src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -611,6 +611,13 @@
             "additionalProperties": false,
             "properties": {
                 "develop": {"type": "boolean", "default": false},
+{% if version >= "3.2" %}
+		"develop_opts":{
+                    "any_gpu": {"type": "boolean", "default": true},
+                    "disable_ibverbs": {"type": "boolean", "default": true},
```

Comment:
Double-negatives (`disable_ibverbs: false`) can get confusing. On the other hand, I do like that "false" consistently means the non-developer behaviour. Possibly it could be called `kernel_networking`, but maybe that doesn't make the connection with ibverbs as clearly. So I'm on the fence.

Reply:
If you are on the fence I'm going to leave it as is then. I agree that the double negatives are confusing but it seems more consistent with the logic of how it was set up originally.